x-pack/test/functional/apps/infra/node_details.ts

```diff
@@ -341,5 +341,59 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         });
       });
     });
+
+    describe('#With Asset Details using nginx host', () => {
+      before(async () => {
+        await Promise.all([
+          esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs'),
+          kibanaServer.savedObjects.cleanStandardList(),
+        ]);
+        await browser.setWindowSize(1600, 1200);
+
+        await navigateToNodeDetails('demo-stack-nginx-01', 'demo-stack-nginx-01');
+        await pageObjects.header.waitUntilLoadingHasFinished();
+      });
+
+      after(async () => {
+        await Promise.all([
```

Comment:
nit: technically no need for `Promise.all()`

Reply:
Good catch, I first started loading/unloading multiple archives and removed them later. Removed

Reply:
Thanks! I also thought it was just a leftover